redbot/cogs/mod/settings.py

```diff
@@ -26,6 +26,7 @@ class ModSettings(MixinMeta):
             respect_hierarchy = await self.settings.guild(guild).respect_hierarchy()
             delete_delay = await self.settings.guild(guild).delete_delay()
             reinvite_on_unban = await self.settings.guild(guild).reinvite_on_unban()
+            default_days = await self.settings.guild(guild).default_days()
```

Comment:
I know this may not be the best PR for it, but fetching the entire guild settings at once via `self.settings.guild(guild).all()` and then unpacking each variable from a simple dict will be less backend-straining than fetching six individual settings that are normally right next to each other.